projects/ngneat/error-tailor/src/lib/control-error.component.ts

```diff
@@ -1,6 +1,12 @@
 import { ChangeDetectionStrategy, ChangeDetectorRef, Component, ElementRef, TemplateRef } from '@angular/core';
 import { ValidationErrors } from '@angular/forms';
 
+export interface IControlErrorComponent {
+  customClass: string;
+  text: string | null;
+  createTemplate?(tpl: TemplateRef<any>, error: ValidationErrors, text: string): void;
```

Comment:
You can type the `TemplateRef` generic.

Reply:
You mean to follow the type you use in your code? 'TemplateRef<{ $implicit: ValidationErrors; text: string }'?

Reply:
Yes